tests_new/integration_tests/modules/s3_datasets/aws_clients.py

```diff
@@ -12,18 +13,34 @@ def __init__(self, session, region):
         self._resource = session.resource('s3', region_name=region)
         self._region = region
 
-    def create_bucket(self, bucket_name, kms_key_id=None):
+    def bucket_exists(self, bucket_name):
```

Comment:
General question here:
Maybe it's better to place all aws_clients into separate folder 'shared' or smth, that we can use in different test-modules?

Reply:
Agree, as soon as other tests use the same AWS clients we could move them outside of each module test to somwhere such as `integration_tests/aws_clients/`